zw-knabbel-wp.php

```
@@ -62,6 +62,232 @@ function debug_enabled(): bool {
+
+			if ( 1 === $updated ) {
+				wp_cache_delete( $option_name, 'options' );
+				wp_cache_delete( 'alloptions', 'options' );
+				wp_cache_delete( 'notoptions', 'options' );
+				return;
+			}
+		}
+
+		usleep( ( $attempt + 1 ) * 1000 );
+	}
+}
```

PHPMD (2.15.0)

[error] The parameter $new_error is not named in camelCase. (undefined)

(CamelCaseParameterName)